app/assets/javascripts/spotlight/pages.js.erb

```diff
@@ -22,8 +22,7 @@ Spotlight.onLoad(function(){
   if (instance.length) {
     var editor = new SirTrevor.Editor({
       el: instance[0],
-      blockTypes:["Heading", "Text", "List","Quote","Iframe", "Video", "Oembed", "Rule", "UploadedItems",
-                 "Browse", "FeaturedPages", "SolrDocuments",  "SolrDocumentsCarousel", "SolrDocumentsEmbed", "SolrDocumentsFeatures", "SolrDocumentsGrid", "SearchResults"],
+      blockTypes: <%= Spotlight::Engine.config.sir_trevor_widgets %>,
```

Comment:
I'm suggesting that we do something in this partial:

https://github.com/projectblacklight/spotlight/blob/dd9805cb240cbc6425c47ea69619b6fd1ed6aa2f/app/views/spotlight/pages/_form.html.erb#L54

Maybe starting with something like:

```
<%= f.text_area_without_bootstrap :content, value: { data: f.object.content.as_json }.to_json, class: 'js-st-instance', data: { blockTypes: Spotlight::Engine.config.sir_trevor_widgets } %>
```

and in the javascript, pull the data from `instance.data('blockTypes')`.